commessa/pom.xml

```diff
@@ -24,6 +24,12 @@
             <version>4.5.14</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
+            <version>1.18.28</version>
+            <scope>provided</scope>
```

Comment:
Perché aggiungere questa dipendenza come provided?
Nell'ambiente target, per il quale in realtà non sono state date specifiche, come fai a dire che è "provided", da chi?